src/main/dynamo/batchGet.ts

```diff
@@ -0,0 +1,49 @@
+import { DynamoDB } from "aws-sdk"
+import { PartitionAndSortKey } from "./keys"
+import { Table } from "./Table"
+import { TaggedModel } from "./types"
+import { UnprocessedKeyCollector } from "./UnprocessedKeyCollector"
+
+export interface BatchGetParams<T extends PartitionAndSortKey<TaggedModel>> {
```

Comment:
This is just extracted from our Beyonce class. In a later commit in this PR, it gets renamed to `batchGetItems`